src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void print() {
+        super.print();
+        System.out.println("circle,"
+                + " area: " + getArea()
+                + " sq.units,"
+                + " radius: "
+                + radius + " sq.units, "
+                + "color: " + getColor());
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
```

Comment:
All magic numbers in your code should be constants. Consider defining 'Math.PI' as a constant if you use it multiple times, or if you want to define precision. If 'Math.PI' is used only here, it's fine as it is widely recognized.